src/tallies/filter_meshsurface.cpp

```diff
@@ -11,7 +11,15 @@ void
 MeshSurfaceFilter::get_all_bins(const Particle& p, TallyEstimator estimator,
                                 FilterMatch& match) const
 {
-  model::meshes[mesh_]->surface_bins_crossed(p, match.bins_);
+  Position r0 = p.r_last_current_;
+  Position r1 = p.r();
+  if (translated_) {
+    r0 -= translation();
+    r1 -= translation();
```

Comment:
Is there a reason why `translation()` is used here, but `translation_` is used in `MeshFilter::get_all_bins`? I'd pick one to keep it consistent

Reply:
Nope! Just forgot to use that accessor method there. Thanks!